custom/succeed/data_sources/patient_task_list.json

```diff
@@ -78,7 +78,7 @@
                             "datatype": "string"
                         },
                         "type": "boolean_expression",
-                        "property_value": "true"
+                        "property_value": "True"
```

Comment:
Nevermind - just saw Cory's link to https://github.com/dimagi/commcare-hq/pull/7783